templates/CRM/Member/Page/DashBoard.tpl

```diff
@@ -30,9 +30,9 @@
     <tr class="columnheader-dark">
       <th scope="col" rowspan="2">{ts}Members by Type{/ts}</th>
         {if $preMonth}
-      <th scope="col" colspan="3">{$premonth} &ndash; {ts}(Last Month){/ts}</th>
+      <th scope="col" colspan="3">{ts}{$premonth}{/ts} &ndash; {ts}(Last Month){/ts}</th>
```

Comment:
You cannot send a variable in "ts", since gettext won't be able to extract the source string.

You might be able to fix upstream by replacing: CRM/Member/Page/DashBoard.php 
$this->assign('premonth', date('F', strtotime($preMonth)));

with:
$this->assign('premonth', CRM_Utils_Date::customFormat($preMonth, '%B'));